src/http.rs

```diff
@@ -125,6 +127,13 @@ async fn handle_schedule(
     Ok(response)
 }
 
+async fn handle_get_recommendations() -> Result<Response<Body>, HttpError> {
+    let nodes = get_recommended_channels().await.unwrap();
+    let mut response = Response::new(Body::from(serde_json::to_string(&nodes).unwrap()));
+    response.headers_mut().insert(hyper::header::CONTENT_TYPE, "application/json".parse().unwrap());
```

Comment:
TODO handle HttpErrors

Reply:
Whats the best way to handle this? 
Match the error and return Error(...)?